src/Symfony/Component/Notifier/Bridge/Ntfy/NtfyOptions.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Ntfy;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+
+/**
+ * @author Mickael Perraud <mikaelkael.fr@gmail.com>
+ */
+final class NtfyOptions implements MessageOptionsInterface
+{
+    private array $options;
+    public const PRIORITY_MAX = 5;
+    public const PRIORITY_URGENT = 5;
+    public const PRIORITY_HIGH = 4;
+    public const PRIORITY_DEFAULT = 3;
+    public const PRIORITY_LOW = 2;
+    public const PRIORITY_MIN = 1;
+
+    public function __construct(array $options = [])
```

Comment:
Constructor property promotion can be used

Reply:
The component is also in my bundle [ntfy-notifier](https://github.com/mikaelkael/ntfy-notifier) available for SF5.4 -> SF6.3. I just adapted the code for the PR. I will modify to use constructor property promotion.